store/src/main/java/com/nytimes/android/external/store3/base/room/RoomPersister.java

```diff
@@ -0,0 +1,33 @@
+package com.nytimes.android.external.store3.base.room;
+
+import com.nytimes.android.external.store3.base.BasePersister;
+
+import javax.annotation.Nonnull;
+
+import io.reactivex.Observable;
+
+/**
+ * Interface for fetching data from persister
+ * when implementing also think about implementing PathResolver to ease in creating primary keys
+ *
+ * @param <Raw> data type before parsing
+ */
+public abstract class RoomPersister<Raw, Parsed, Key> implements RoomDiskRead<Parsed, Key>, RoomDiskWrite<Raw, Key>, BasePersister {
```

Comment:
Parsers are unnecessary since the DB read query can be a "parsed" version of whatever the written "raw" data is